addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -21,7 +21,7 @@ forced in the base s_masonry_block at the moment.
 
 <template id="s_masonry_block_default_template" name="Masonry">
     <t t-snippet-call="website.s_masonry_block">
-        <div class="row o_grid_mode" data-row-count="8" style="gap: 16px;">
+        <div class="row o_grid_mode o_snippet_not_selectable" data-row-count="8" style="gap: 16px;">
```

Comment:
This is not the right solution. I really don't know how I/we missed this while merging the masonry "exploding" task but the problem here is that the `data-snippet` attribute is automatically added on this node... this is not right. The main node of the snippet is there thanks to `t-snippet-call` and receives the `data-snippet="s_masonry_block"` as excepted... "s_masonry_block_default_template" should not appear a few nodes below.

Hint: _compile_directive_snippet